### DISCUSSION TITLE
Catalog Studio Gallery Previews - Coming soon.

### DISCUSSION THREAD

**Discussion by MattB-msft** [microsoft/PowerPlatform-Catalog#22] [Announcements]
As some folks know, we are starting the private preview for Catalog Gallery in the Power* Studios

A few important notes: 

- We have not yet configured the Catalog Managers to automatically update. 
- As part of this preview, it's important to keep your Catalog Managers up to date to make sure you are getting the latest features as we respond to feedback.   
- If you're setting up Auto Approval or want to configure auto approval, please review [this](https://github.com/microsoft/PowerPlatform-Catalog/wiki/AutoApproval) wiki article 
- If you have questions, please feel free to log an issue on the issues board

**Comment by fmgel22** [comment]
may i use a copilot studio agent to check solutions available in the catalog? if yes, can you help me to know how? i coudn't find Dataverse tables to let CS agent to check
